src/app/cli/src/tests/coda_worker.ml

```diff
@@ -456,7 +456,8 @@ module T = struct
           in
           let monitor = Async.Monitor.create ~name:"coda" () in
           let with_monitor f input =
-            Async.Scheduler.within' ~monitor (fun () -> f input)
+            Async.Scheduler.within' ~monitor ~priority:Priority.low (fun () ->
```

Comment:
This change is probably unnecessary, this is only for the old integration tests.